app/main.py

```
@@ -0,0 +1,26 @@
+
+from app.api.auth import router as auth_router
+
+
+app = FastAPI(title="FitConnect API")
+
+origins = [
+    "*"  # 개발 중에는 모두 허용, 배포시에는 프론트 도메인만 넣기
+]
+
+app.add_middleware(
+    CORSMiddleware,
```

Ruff (0.13.1)

Undefined name `CORSMiddleware`

(F821)